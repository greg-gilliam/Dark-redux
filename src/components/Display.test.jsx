import { screen, render } from "@testing-library/react";
import Display from "./Display";

it("should render the Display component", () => {
  const { container } = render(<Display catchphrases={[]} />);
  expect(container).toMatchSnapshot();
});
