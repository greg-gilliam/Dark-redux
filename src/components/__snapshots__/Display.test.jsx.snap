// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Display component 1`] = `
<div>
  <div />
</div>
`;
